app/src/main/java/net/devemperor/dictate/core/DictateInputMethodService.java

```diff
@@ -927,6 +941,14 @@ private void startGPTApiRequest(PromptModel model) {
     }
 
     private void sendLogToCrashlytics(Exception e) {
+        // Temporarily disable Crashlytics since Firebase is not configured
+        // Log the error to console instead
+        StringWriter sw = new StringWriter();
+        e.printStackTrace(new PrintWriter(sw));
+        Log.e("DictateInputMethodService", sw.toString());
+        Log.e("DictateInputMethodService", "Error logged (Crashlytics disabled)");
```

Comment:
These changes were just for my own quick tessting; These lines should be reverted before a merge.